client/src/service.rs

```diff
@@ -65,22 +65,22 @@ where
 {
     type Command = p2p::Command;
 
-    fn command(&mut self, cmd: Self::Command) {
+    fn command_received(&mut self, cmd: Self::Command) {
         // TODO: Commands shouldn't be handled by the inner state machine.
         self.machine.command(cmd)
     }
 }
 
-impl<T, F, P, C> StateMachine for Service<T, F, P, C>
+impl<T, F, P, C> PeerProtocol for Service<T, F, P, C>
```

Comment:
```suggestion
impl<T, F, P, C> PeerProtocol for Service<T, F, P, C>
```

Maybe `PeerProtocolState` is more explicative for a newcomer in the codebase?

Reply:
It is not that much about state as much about API for hooks on network operations